examples/webcrawler/src/main/java/com/theagilemonkeys/webcrawler/config/AppConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package com.theagilemonkeys.webcrawler.config;
+
+import io.github.cdimascio.dotenv.Dotenv;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.nio.file.Paths;
+
+@Configuration
+public class AppConfiguration {
+    @Bean
+    public EmbeddingsSpaceConfiguration embeddingsSpaceComponent() {
+        String path = Paths.get("examples","webcrawler", ".env").toString();
+        Dotenv dotenv = Dotenv.configure()
+                .directory(path)
+                .filename(".env")
+                .load();
+        return new EmbeddingsSpaceConfiguration(
```

Comment:
If you're using springboot, you don't need a `.env.example` file to inject these values here. You can use the `application.properties` file under the resources folder, you can check an example here -> https://docs.ellmental.com/getting-started#ellmental-%EF%B8%8F-springboot